editoast/src/modelsv2/infra.rs

```diff
@@ -60,6 +64,7 @@ pub struct Infra {
     #[serde(skip)]
     pub owner: Uuid,
     pub version: String,
+    #[schema(required)]
```

Comment:
This is a model so it will only appear in responses, and since `None` serializes to `null`, we instruct the frontend that the field will always be there, even if null. This way the frontend get slightly better types (`string | null` instead of `string | null | undefined`).